### ISSUE TITLE
Strange tagging for some verb forms of "sein" and for "du"

### ISSUE THREAD

**Issue by CJAnti** [markuskiller/textblob-de#9]
I stumbled over some strange tagging and was wondering why it won't correctly recognize "bist" and "seid" as verb forms of "sein", though they are listed in the "de-verbs.txt" file. Also tagging the personal pronoun "du" as an adjective doesn't make much sense either.

```
>>> blob=TextBlob("Ich bin. Du bist. Er ist. Wir sind. Ihr seid. Sie sind.",
                  parser=PatternParser(pprint=True, lemmata=True))
>>> blob.parse()
          WORD   TAG    CHUNK   ROLE   ID     PNP    LEMMA   

           Ich   PRP    NP      -      -      -      ich     
           bin   VB     VP      -      -      -      sein    
             .   .      -       -      -      -      .       
            Du   JJ     NP      -      -      -      du      
          bist   NN     NP ^    -      -      -      bist    
             .   .      -       -      -      -      .       
            Er   PRP    NP      -      -      -      er      
           ist   VB     VP      -      -      -      sein    
             .   .      -       -      -      -      .       
           Wir   PRP    NP      -      -      -      wir     
          sind   VB     VP      -      -      -      sein    
             .   .      -       -      -      -      .       
           Ihr   PRP$   NP      -      -      -      ihr     
          seid   NN     NP ^    -      -      -      seid    
             .   .      -       -      -      -      .       
           Sie   PRP    NP      -      -      -      sie     
          sind   VB     VP      -      -      -      sein    
             .   .      -       -      -      -      .     
```

**Comment by mk270** [comment]
By the way, do you know whether rftagger is open source?

**Comment by markuskiller** [comment]
@mk270 rftagger is open-source and its source code is available under the following links:

project page: http://www.cis.uni-muenchen.de/~schmid/tools/RFTagger/

source code: http://www.cis.uni-muenchen.de/~schmid/tools/RFTagger/data/RFTagger.tar.gz

However, it is "freely available for **education, research and other non-commercial purposes" only**. If this is not a problem for your project, feel free to contact me via email for me to be able to invite you to the bitbucket repository of `textblob-rftagger`. It is **fully functional**, the only reason I'm holding it back is because I haven't had the time to sort out a sensible and secure way of distributing the included binaries and because I'm unsure about the licensing concerning these binaries.

**Comment by mk270** [comment]
So, "available for education, research and other non-commercial purposes" is fairly canonically NOT open source, see for instance http://opensource.org/osd-annotated section 6.

It's a shame. I presume, since they've chosen to exclude commercial use, that they're not going to be biddable.

**Comment by markuskiller** [comment]
Thanks for the link. I interpreted 'open source' as 'is the source code available/accessible' (i.e. can it be modified/tweaked, etc.), which it is.

On other projects they released under a similarly restrictive license, they added:

"In order to use the TreeTagger commercially, you need to obtain a commercial license (see contact address below)! " (Source: http://www.cis.uni-muenchen.de/~schmid/tools/TreeTagger/). So, I assume that they might be willing to make a decision/offer/quote on a per project basis and that they want to know what the project is about if you intend to use their software commercially.

**Comment by mk270** [comment]
Yes, indeed, it's not remotely open source in that term's conventional acceptation - it's proprietary.

I am asking as it's a dependency of another project I'm interested in. Ah well.
